dist/scripts/rx-data-table.js

```diff
@@ -406,16 +406,29 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
                 }, errorDisplayTimeout);
             });
 
-            scope.scoreTooltipTemplate = function (score) {
+            scope.scoreTooltipTemplate = function (row) {
                 var template = [
-                    'Waiting on Racker for ', $filter('rxAge')(score.lastResponseTime, 3, true)
+                    'Waiting on Racker for ', $filter('rxAge')(row.score.lastResponseTime, 3, true),
+                    'Score: ',
+                    '-(',
+                    row.score.weight,
```

Comment:
@JoseVillarreal: Do they just want to see the numbers or do they also need to know what those numbers mean ?